esp/templates/users/usersearch/list_selector.html

```diff
@@ -73,6 +73,14 @@
             <div class="step_instructions step_highlight">Currently selected: <span id="filter_current_list"></span></div>
             <div id="filter_accordion">
             
+                <h4><a href="#">User Group</a></h4>
```

Comment:
Let's put this down at the end, or at least further down -- I think it will be confusing to add something new at the top.  I'd put it either below email or at the end.